src/main/resources/alfresco/templates/webscripts/org/alfresco/repository/solr/transactionInterval.get.json.ftl

```diff
@@ -0,0 +1,4 @@
+{
+    "minTransactionCommitTimeMs": <#if minTransactionCommitTimeMs??>${minTransactionCommitTimeMs?c}<#else>-1</#if>,
+    "maxTransactionCommitTimeMs": <#if maxTransactionCommitTimeMs??>${maxTransactionCommitTimeMs?c}<#else>-1</#if>
```

Comment:
Why would you want to specify max txn, it should always be listening to changes that may fall in the db id range

Reply:
This web script is used by DB_ID_RANGE Sharding method, so it's also worth to skip new transactions related with nodes that are out of the Shard range.

Reply:
This API just returns it (I presume for symmetry), but you're right that it shouldn't be used when skipping transactions as more nodes may appear in the range later on.

Reply:
Sample

* Shard 1 configured with DB_ID_RANGE from 0 to 800 
* minTransaction is 1 and maxTransaction is 1000

There are new transactions (from 1000 to 2000) that are not related with nodes 0 to 800. If you know that your maxTransaction is 1000, you can ignore every transaction from 1001 to 2000. This is how maxTransaction value is used.

Reply:
This works for an initial ingest, and as long as you know the end transaction id of the ingest (2000 in your example). After this point then there might be transactions containing updates to existing nodes which are relevant to the shard.

Reply:
This is the other condition, right. As maxTransaction is compared with lastTransactionIndexed in every shard. So if maxTransaction > lastTransactionIndexed, it will be indexed. But if maxTransaction < lastTransactionIndexed, it won't be indexed.